src/hotspot/os_cpu/linux_zero/os_linux_zero.cpp

```diff
@@ -288,16 +386,27 @@ size_t os::current_stack_size() {
 /////////////////////////////////////////////////////////////////////////////
 // helper functions for fatal error handler
 
-void os::print_context(outputStream* st, const void* context) {
-  ShouldNotCallThis();
+void os::print_context(outputStream* st, const void* ucVoid) {
+  st->print_cr("No context information.");
```

Comment:
Regrettable, maybe something for a future RFE?

Reply:
Yes. I don't want to copy-paste the whole arch-specific code into this file. The bare minimum to have reasonable `hs_err`-s should do :)

Reply:
I forgot to mention why I do this. If we leave `ShouldNotCallThis()` here, then error handler does a secondary fault in these methods, and `hs_err` prints errors in these sections. So the deal here is removing the risky secondary fault -- I see no reason for it -- rather than printing out more info. When we can do that easily, like in the TOS and PC sections, we do.

Reply:
Sure, makes sense to avoid them. Secondary crashes also build up stack, therefore they are limited to only a few before error handling just stops.